girder_annotation/girder_large_image_annotation/models/annotationelement.py

```diff
@@ -374,6 +415,23 @@ def _boundingBox(self, element):
             bbox['highx'] = max(x0, x1)
             bbox['highy'] = max(y0, y1)
             bbox['details'] = len(element['values'])
+        elif element.get('type') == 'imageoverlay':
+            lowx = highx = lowy = highy = highz = 0
+            try:
+                lowx, highx, lowy, highy = Annotationelement()._overlayBounds(element)
+                baseImageItem = ImageItem().load(annotation['itemId'], force=True)
+                baseImageMetadata = ImageItem().getMetadata(baseImageItem)
+                highz = baseImageMetadata.get('levels', 0)
+            except Exception as e:
+                logger.error('Error generating bounding box for image overlay annotation: %s' % e)
```

Comment:
I think it would be better to have this guard in the `_overlayBounds` method rather than here.

Reply:
See 95cf345